hangman.rb

```diff
@@ -0,0 +1,184 @@
+require "Colorize"
+
+class Hangman
+
+  def initialize
+
+    #this runs at the beginning and calls lots of other methods
+
+    # initialized variables
+    @answer = pick_answer
+    # User has 7 wrong guesses before game over
+    @no_of_guesses = 0
+    @blanks_array = answer_to_blanks
+    @guessed_letters = [ ]
+
+    # methods we need to call in the flow of the game
+    start
+    art
+    get_guess
+
+  end
+
+  def start
+    puts "Welcome to Hangman! Type " + "'quit'".colorize(:red) + " at any time to exit."
+  end
+
+
+# creates an answer from the array that is randomized
+  def pick_answer
+    # randomly pick the answer from this array
+    library = ["cat", "dog", "mouse", "letter", "elephant",
+      "bottomless", "house", "polecat", "coffee", "apple",
+      "laptop", "caramel", "sushi", "computer", "television"]
+    @answer = library.sample
+    return @answer
+  end
+
+  def answer_to_blanks
+    # convert the @answer to same number of blanks
+    # so we can display that clue for the user
+    num_of_blanks = @answer.length
+    @blanks = "_" * num_of_blanks
+    puts @blanks
+
+    # split up the blank array into individual "_"  strings
+    # so we can change each one individually
+    @blanks_array = @blanks.split(//)
```

Comment:
Same comment here about the `@blank_array` instance variable. Since you're returning it below, you really should use a local variable here and return it and then set it using an instance variable. Returning it and using the instance variable here is redundant.